manifests/qpid.pp

```diff
@@ -18,12 +18,24 @@
     require   => Service['qpidd'],
     logoutput => true,
   } ~>
-  exec { 'bind katello entitlements queue to qpid exchange messages that deal with entitlements':
-    command   => "qpid-config --ssl-certificate ${client_cert} --ssl-key ${client_key} -b 'amqps://localhost:5671' bind event ${candlepin_event_queue} '*.*'",
-    onlyif    => "qpid-config --ssl-certificate ${client_cert} --ssl-key ${client_key} -b 'amqps://localhost:5671' queues ${candlepin_event_queue}",
-    path      => '/usr/bin',
-    require   => Service['qpidd'],
-    logoutput => true,
+  qpid::bind_event { 'entitlement.created':
```

Comment:
You could even simplify this:
```puppet
qpid::bind_event { ['entitlement.created', 'entitlement.deleted', 'pool.created', 'pool.deleted', 'compliance.created']:
  ssl_cert => $client_cert,
  queue    => $candlepin_event_queue,
}
```

That would even allow you to parametrize which events to bind to.

Now I am wondering (and we didn't catch this in the other PR): where did the ```--ssl-key ${client_key}``` parameter go?